internal/repository/transaction_repo.go

```diff
@@ -10,11 +10,13 @@ import (
 
 var (
 	ErrTransactionNotFound = errors.New("transaction not found")
-	//ErrMissingParams   = errors.New("must provide a name and last name for an account")
+	ErrMissingFields       = errors.New("transaction is missing one or more of these: owner, sender, receiver, amount")
+	ErrZeroAmount          = errors.New("transaction amount cannot be zero")
 )
 
 type TransactionRepo interface {
 	FindAll(ctx context.Context) ([]models.Transaction, error)
+	FindOne(ctx context.Context, id string) (models.Transaction, error)
```

Comment:
We should be adding the Create to the interface, so we can use it later on the service layer